input-profile.js

```diff
@@ -0,0 +1,194 @@
+import {Component, Type} from '@wonderlandengine/api';
+import {vec3, quat} from 'gl-matrix';
+
+export class InputProfile extends Component {
+    static TypeName = 'input-profile';
+    static Properties = {
+        controller: {type: Type.Object},
+        handednessIndex: {
+            type: Type.Enum,
+            values: ['left', 'right'],
+            default: 'left',
+        },
+        path: {
+            type: Type.String,
+            default:
+                'https://cdn.jsdelivr.net/npm/@webxr-input-profiles/assets@1.0.13/dist/profiles/',
+        },
+        customProfileFolder: {
+            type: Type.String,
+        },
+    };
+
+    init() {
+        this.gamepadObjects = {};
```

Comment:
Private variable should be prefixed with an undescore. I dont know if any of those is private